### PR TITLE
Improvement: No eslint errors left

### PR DESCRIPTION
**Pull request by zelibobla**
`eslint src/app/**` is now ready to be launched in CI pipeline

### CODE REVIEW

File: lib/src/app/helpers/createPageLevelParameters.js [hunk 1]
```diff
@@ -2,6 +2,12 @@ import defaultSkin from '../skin';
 import defaultsDeep from 'lodash.defaultsdeep';
 import get from 'lodash.get';
 
+/**
+ * Performs complicated merge of default and custom config objects
```
Comment (aeng7) [inline lib/src/app/helpers/createPageLevelParameters.js]: Can you expand on "complicated" in the jsdoc?

**Comment by aeng7** [comment]
LGTM other than a minor comment

File: lib/src/app/idleTimer.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ import * as logger from 'loglevel';
 import castManager from './receiverManager';
 
 const timeout = {
-  PAUSED: 1000 * 60 * 5, // 5 minutes
-  IDLE: 1000 * 60 * 10, // 5 minutes
+  PAUSED: 1000 * 60 * 5, // eslint-disable-line no-magic-numbers
```
[the remark this answers is not in the document]
Comment (zelibobla) [reply lib/src/app/idleTimer.js]: Not wrong, but I've returned back an explanation.

File: lib/src/app/idleTimer.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ let idleTimerId = null;
  * setIdle destroy any current timeout and creates a new one with the
  * given time
  *
- * @param {number} time
+ * @param {number} time The time after which castManager should get idle
```
[the remark this answers is not in the document]
Comment (zelibobla) [reply lib/src/app/idleTimer.js]: done

File: lib/src/app/helpers/createPageLevelParameters.js (hunk 1 above)
Comment (zelibobla) [reply lib/src/app/helpers/createPageLevelParameters.js]: done

**Pull request merged**